src/modules/esl-toggleable/core/esl-toggleable.ts

```diff
@@ -235,6 +253,10 @@ export class ESLToggleable extends ESLBaseElement {
   protected onHide(params: ESLToggleableActionParams): void {
     CSSClassUtils.remove(this, this.activeClass);
     CSSClassUtils.remove(document.body, this.bodyClass, this);
+    if (this.containerActiveClass) {
+      const $container = findParent(this, this.containerActiveClassTarget);
```

Comment:
mb can we move the declaration of this variable?

Reply:
So you meant to have a 
```ts
@memoize
get $containerActiveClassTarget(): HTMLElement | null {
  return findParent(this, this.containerActiveClassTarget);
}
```

That's reasonable in case we want to roll back classes if the attribute value has changed.
But we need to extend attributeChangeCallback properly in that case.

@exadel-inc/esl-all-contributors , 
do we need to clean-up classes (at least positive ones) for a container in case it has changed ?

Reply:
ToDO: @ala-n to create as a separate issue